src/cai_lab_to_nwb/zaki_2024/zaki_2024_convert_session.py

```diff
@@ -40,6 +40,7 @@ def session_to_nwb(
     include_sleep_classification: bool = True,
     include_behavioral_video: bool = True,
     include_eeg_emg_signals: bool = True,
+    include_shock_times: bool = False,
 ):
     print(f"Converting session {session_id}")
```

Comment:
False by default? 
Also, shouldn't the print in line 45 should be be under verbose?